src/app/multi-line-chart/multi-line-chart.component.ts

```diff
@@ -227,6 +227,28 @@ export class MultiLineChartComponent implements OnInit {
           )
       );
 
+      // x axis label
+      svg
+        .append('text')
+        .attr('font-family', 'sans-serif')
+        .attr('font-size', 10)
+        .attr('y', self.height)
+        .attr('x', self.width - 10)
+        .attr('text-anchor', 'end')
+        .attr('font-weight', 'bold')
+        .text(data.x);
```

Comment:
I removed these in #7 because they were previously aligned to the last tick mark (and I had modified tick mark logic so the last tick mark might not align with the edge of the graph) and I didn't know how to make them aligned to the graph instead of the last tick mark

But it's pretty easy of course :)

<img width="523" alt="Screen Shot 2020-04-16 at 3 49 36 PM" src="https://user-images.githubusercontent.com/3111845/79513953-ed1e6d80-7ff9-11ea-9d6e-2928b5f0f89b.png">